tempest/api/workloadmgr/barbican/test_barbican.py

```diff
@@ -327,25 +330,30 @@ def test_01_barbican(self):
                 self.vm_id, "/opt/File_4")
             snapshot_wise_filecount = self.verifyFilepath_Search(
                 filesearch_id, "/opt/File_4")
-            for snapshot_id in filecount_in_snapshots.keys():
-                if snapshot_wise_filecount[snapshot_id] == filecount_in_snapshots[snapshot_id]:
-                    filesearch_status = True
-                else:
-                    filesearch_status = False
-
-            if filesearch_status:
-                LOG.debug("Filepath_Search default_parameters successful")
+            if(snapshot_wise_filecount == None):
```

Comment:
In what case is  None returned by verifyFilepath_Search() ?

Reply:
Removed this code. Not required. It was setup issue and was not needed to be handled.